### ISSUE TITLE
Training Parameters

### ISSUE THREAD

**Issue by Cre1th** [fizyr/keras-retinanet#213]
How many epochs where run for the results presented? Is this the default 50?

Is there any reason why you do not use `keras.callbacks.EarlyStopping` to prevent overfitting? This could be coupled with the COCO evaluation metric.

**Comment by hgaiser** [comment]
I think it ran for nearly twice that, so 100 epochs. Not sure though, was a while back that I ran that.

I manually check which epoch gave the best COCO score, since at the end it tends to oscillate a little bit around a certain optimum.

Since this isn't really an issue, I will close this. If you have further questions, feel free to join the Slack channel (check the README for description on how to join).
